docs/docs/examples/guides/ur-sim-guide.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 5
+title: URSim Guide
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import ursimActivate from './assets/ursim-activate.gif'
+import ursimHardwareParams from './assets/ursimHarwareParams.png'
+
+
+# URSim guide
+
+This guide aims to support users with running URSim and connecting it to AICA Studio. URSim is a useful simulation
+environment that provides access to a virtual robot. Users can utilize this environment to get familiar with the 
+robot interface, as well as create and run their own programs. Please note that compared to the real robot, 
+functionality that is not present includes:
+
+- Emergency stop button
+- Input IO state
+- Self or surrounding object collisions
+- Force mode
+
+For more information, please refer to the [installation page](https://www.universal-robots.com/download/software-ur-series/simulator-linux/offline-simulator-ur-series-e-series-ur-sim-for-linux-5220/). 
+
+TBD: Should we keep the following note?
+
+:::note
+URSim is made for Linux. For other operating systems, a virtual machine is needed. For more information, check the 
+[official installation instructions](https://www.universal-robots.com/download/software-ur-series/simulator-non-linux/offline-simulator-ur-series-e-series-ur-sim-for-non-linux-5220/). 
+:::
+
+
+## Installation
+
+Executing the following instructions runs URSim in a Docker container.
+
+1. Clone the `simulators` repository provided by AICA. 
+
+```bash
+git clone https://github.com/aica-technology/simulators.git
+```
+
+TBD: must have Docker installed. Should we add something about that?
+Probably people reading this have already done that. 
+
+2. Navigate to the `ursim` folder inside this repository. 
+
+3. Run the `run.sh` script.
+
+```bash
+./run.sh
```

Comment:
Here we might need to specify a few more things in the script (this is not the correct line to introduce this, I'm just typing my comment here)

```suggestion
./run.sh -v 5.16.1 -m ur5e
```

v5.2x is still good for all our remote things, but is technically incompatible with some UR scripts from earlier versions. Just for the sake of being sure I think we should somehow mention this (perhaps in a note block).

We don't have to explicitly say 5.16.1, but we just need to mention the incompatibilities between 5.1x and 5.2x

fyi @domire8

Reply:
really? what kind of incompatibilities?

Reply:
move commands are now using optimove and for whatever reason this seems to create some kind of incompatibility where applications built in newer version are not backwards compatible (never looked too much in depth, but that's the jist of it)

Reply:
loading .urp files you mean? Does UR have a page where this is mentioned? Otherwise I'd rather not go into explaining the incompatibilities but rather just saying that "you can use the same version as on your robot by providing the argument"